lib/theme_check/template.rb

```diff
@@ -58,13 +58,20 @@ def root
       parse.root
     end
 
-    def update!(line, value)
-      lines[line - 1] = value
-      @source = lines.join("\n")
+    def corrected_lines
```

Comment:
Can we keep the "corrected" word out of `Template`? I don't think it needs to have any knowledge of autocorrection.

And technically they are not really corrected, unless `correct_line` has been called before.

I would rename to `lines`, and rename `correct_line` to `update_line`.

Reply:
Ah for this I didn't want to amend the existing `lines` ivar in template, instead creating a duplicate that holds the amended lines. Although for now I don't see a problem amending lines directly, it feels a little iffy to dirty the source of truth. 

Lemme work through this naming, although would love to get your thoughts on amending the lines ivar directly